### DISCUSSION TITLE
One global file with all recipients

### DISCUSSION THREAD

**Discussion by gawsoftpl** [FiloSottile/age#521] [UX reports]
Age is cool, but I am looking for something like this:

Setup one global file for example in 
/home/user/.age-recipients where will be stores all public keys to encrypt data.

When I encrypt files 
/home/user/data/test.bin
/home/user/data/test2.bin/

age will get all recipients from .age-recipients. I cant find any info about that, age can autodetect config file in the global folder.

**Comment by Turmaxx** [comment]
You can export the path of the receipents you just mentioned in to shell's .rc for now that is. 

But the point of age is that is that doesn't need configuration.
